server/src/main/java/info/doseamigos/meds/MedService.java

```diff
@@ -0,0 +1,15 @@
+package info.doseamigos.meds;
+
+/**
+ * Service for handling all business logic associated with Meds.
+ */
+public interface MedService {
+
+    /**
+     * Adds a new medication to the user's list by looking it up by name in the external api, then populating the
+     * AmigoUser and rxcui afterward.
+     * @param name The Name of the med to look up.
+     * @return A new Med object for the user with the name.
+     */
+    Med addByName(String name);
```

Comment:
My thinking is that the user only supplies a name to Alexa, so we need to do a dynamic lookup of that medication by it's name.  Eventually, we might need to display options on the cards for spelling suggestions if one cannot be found.